src/LegacyStrings.jl

```diff
@@ -49,15 +49,15 @@ using Compat
     end
 
     if VERSION >= v"0.5.0-"
-        immutable ASCIIString <: DirectIndexString
+        struct ASCIIString <: DirectIndexString
```

Comment:
There have been no changes to master since the last release, so a new tag wouldn't change anything. The issue is that any bugs that are found or issues that need fixing would have to be backported to a branch that supports older Julia versions, which is quite annoying.